packages/backend/package.json

```diff
@@ -45,6 +45,7 @@
     "@immobiliarelabs/backstage-plugin-gitlab-backend": "6.2.0",
     "@janus-idp/backstage-plugin-keycloak-backend": "1.5.3",
     "@janus-idp/backstage-plugin-ocm-backend": "3.2.1",
+    "@janus-idp/backstage-plugin-aap-backend": "^1.2.2",
```

Comment:
I think we should pin the dependency similar to how we're dealing with all the other dependencies